.github/workflows/dapr.yml

```diff
@@ -687,3 +687,38 @@ jobs:
             echo "::error::There is no change for ${daprVersion} Helm chart. Did you forget to update the chart version before tagging?"
             exit -1
           fi
+
+  update-longhauls:
+    name: Update the dapr version in longhaul tests
+    runs-on: ubuntu-latest
+    needs: helmpublish
+    if: startswith(github.ref, 'refs/tags/v') && github.repository_owner == 'dapr'
+    steps:
+      - name: Get Token
+        id: get_workflow_token
+        uses: peter-murray/workflow-application-token-action@v2
+        with:
+          application_id: ${{ secrets.LONGHAUL_APPLICATION_ID }}
+          application_private_key: ${{ secrets.LONGHAUL_APPLICATION_PRIVATE_KEY }}
```

Comment:
https://github.com/actions/create-github-app-token
I think it'd be worth throwing this official action into the mix for consideration

Reply:
Yep, done! I had automatically gone with the one I knew from another project, but official is better, for sure!